app/gui/src/controller/searcher/action.rs

```diff
@@ -156,17 +156,31 @@ pub struct Subcategory {
 // === List Entry ===
 // ==================
 
+/// Which part of the component browser entry was best matched to the searcher input.
+#[derive(Clone, Debug, Default)]
+pub enum MatchKind {
+    /// The entry's action was matched.
+    #[default]
+    Action,
```

Comment:
I don't understand what is "entry's action" and how it differs from label or code.

Reply:
I wasn't sure how action list entries were matched to the searcher input, I've changed those matches to `Code` now.